sdk/src/types/block/payload/milestone/option/mod.rs

```diff
@@ -92,6 +101,17 @@ impl MilestoneOptions {
         Ok(Self(milestone_options))
     }
 
+    /// Creates a new [`MilestoneOptions`] from an ordered set.
+    pub fn from_set(milestone_options: BTreeSet<MilestoneOption>) -> Result<Self, Error> {
```

Comment:
Does that not need a specific Ord impl or something? How are they ordered?

Reply:
Surprised the compiler didn't catch this, thanks.

Reply:
I'm still not sure how this is compiling tbh

Reply:
Does `BTreeSet` not have an Ord bound?

Reply:
No, only the fns do.